cloud_cleaner/resources/server.py

```diff
@@ -36,16 +38,27 @@ def register(self, config):
         _desc = "Minimum age (1d, 2w, 6m, 1y)"
         self._sub_config.add_argument("--age", "-a", help=_desc)
 
-    def process(self):
+    def process(self, deletion):
         """
         Fetches the list of servers and processes them to filter out which
         ones ought to actually be deleted.
 
         :return: None
         """
-        shade = self._get_shade()
+        conn = self._get_conn()
         self._config.info("Connecting to OpenStack to retrieve server list")
-        self.__targets = shade.list_servers()
+        self.__age = self._config.get_arg('age')
+        self.__targets = conn.list_servers()
+        if(not deletion):
```

Comment:
Parenthesis are out of place in Python when not needed for grouping or spacing across multiple lines.